base/lnls-prune-artifacts.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+set -eu
+
+target_paths=$@
+
+
+get_used_epics_modules() {
+    targets=$@
+
+    executables=$(find $targets -type f -executable -print)
+    libs="$(ldd $executables 2>/dev/null)"
+
+    modules=()
+    epics_libs=$(echo "$libs" | grep -oE "$EPICS_MODULES_PATH/.*/*.so")
+    for lib in $epics_libs; do
+        module=${lib%/lib/*/*lib*so}
```

Comment:
Use `epics_lib` or even something other than `lib`. Otherwise the substitution gets confusing with the repeated `lib`, and it's confusing enough already.

Add a comment explaining the purpose of expression (an example would be enough).

Reply:
Let me know what you think about the comment.